client/app/api/course/Forum/Forums.ts

```diff
@@ -23,6 +23,7 @@ export default class ForumsAPI extends BaseCourseAPI {
    * Fetches an array of forums.
    */
   index(): APIResponse<{
+    forumTitle: string | null;
```

Comment:
Preferably just ensure backend responds with string type. Don't deal with mixed types.

Reply:
if we proceed with the string type (means the null value will be treated as ''), the logic for determining the title page will be complicated (instead of merely `forumTitle ?? t(translation.header)`, we need to do it like this `forumTitle === '' ? t(translation.header) : forumTitle`..

Reply:
`forumTitle || t(translation.header)` will work right?